frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -176,9 +179,20 @@ export const personsLogic = kea<personsLogicType<PersonFilters, PersonLogicProps
                     action = parsedValue !== undefined ? 'updated' : 'removed'
                 }
 
-                actions.setPerson(person) // To update the UI immediately while the request is being processed
-                const response = await api.update(`api/person/${person.id}`, person)
```

Comment:
Confused - so the old endpoint still can get person model out of sync? I don't think we've solved the problem then.

Reply:
I'm going step by step here - first updating the UI to behave correctly (which is the main way people use this). I'll then yeet whatever code remains once I map out all the implications.

Note I'm on support hero time so am shipping in increments

Reply:
Are you planning on yeeting this in this PR (is this ready for a review)?

If not, then I can't really approve this without e.g. a ticket indicating the problem.

I'm not sold on this approach either - why not change the original endpoint?

Reply:
This is ready for a review - it updates the path for updating a person's property via the UI.

The original endpoint processes a full patch to the person, and I need to make sure I'm aware of what depends on it. It is a more general endpoint where people can patch not just properties. 

One issue at a time.

Reply:
> One issue at a time.

We're promoting our API more and more and this fix doesn't fix the existing API. We should either:
1. Fix this issue properly in this PR
2. Or create a separate issue for that issue 

That's what I was asking for in the original comment - if you're going for (2) please link that ticket. Without this I can't accept this fix as we're introducing needless pain for ourselves in the future.

Reply:
https://github.com/PostHog/posthog/issues/9856

Reply:
Thank you!